### PR TITLE
[MTDSA-13690] [CK] Update readme.md with changelog section and archive link to previous cha…

### PR DESCRIPTION
**Pull request by sudoku007**
…ngelog wiki.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
+# Income Tax (Making Tax Digital) Changelog
 
-# income-tax-mtd-changelog
 
-The following Income Tax MTD services are supported:
+### List of services
 
+The changelog of the following Income Tax MTD services are supported:
```
Comment (mattstephens-hmrc) [inline README.md:6]: If we're keeping the list of services, could this be phrased something like:
"This page contains the latest changes for the following Income Tax MTD API services:"

File: README.md [hunk 2]
```diff
@@ -21,10 +22,16 @@ The following Income Tax MTD services are supported:
...
 
-### Changelog
 
-You can see our changelog [here](https://github.com/hmrc/income-tax-mtd-changelog/wiki)
+### Changelog updates
+
+
+### Previous changelog updates
+
+[Archive of previous changelog updates available here](https://github.com/hmrc/income-tax-mtd-changelog/wiki)
```
Comment (mattstephens-hmrc) [inline README.md:33]: It doesn't need the "available here", the link name can just be the name of the thing it's pointing to

File: README.md [hunk 3]
```diff
@@ -21,10 +22,16 @@ The following Income Tax MTD services are supported:
...
 * [individuals-business-eops-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-business-eops-api/1.0)
+* [self-employment-business-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-employment-business-api/1.0)
 
-### Changelog
 
-You can see our changelog [here](https://github.com/hmrc/income-tax-mtd-changelog/wiki)
+### Changelog updates
+
+
+### Previous changelog updates
```
Comment (mattstephens-hmrc) [inline README.md:31]: This could just say "Previous updates"

File: README.md [hunk 4]
```diff
@@ -21,10 +22,16 @@ The following Income Tax MTD services are supported:
 * [cis-deductions-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/cis-deductions-api/1.0)
 * [individuals-charges-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-charges-api/1.0)
 * [individuals-business-eops-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-business-eops-api/1.0)
+* [self-employment-business-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-employment-business-api/1.0)
 
-### Changelog
 
-You can see our changelog [here](https://github.com/hmrc/income-tax-mtd-changelog/wiki)
+### Changelog updates
```
Comment (mattstephens-hmrc) [inline README.md:28]: This could just say "Changelog", the "updates" is redundant

File: README.md [hunk 5]
```diff
@@ -1,8 +1,9 @@
+# Income Tax (Making Tax Digital) Changelog
 
-# income-tax-mtd-changelog
 
-The following Income Tax MTD services are supported:
+### List of services
```
Comment (mattstephens-hmrc) [inline README.md:4]: Doesn't really need the heading as it's the introduction

File: README.md (hunk 1 above)
Comment (sudoku007) [reply README.md]: Updated as per feedback and pushed the changes

File: README.md (hunk 5 above)
Comment (sudoku007) [reply README.md]: Updated as per feedback, can you review again?

**Review by mattstephens-hmrc — APPROVED** [review]

**Pull request merged**